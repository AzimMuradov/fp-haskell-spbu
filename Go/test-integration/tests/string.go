func main() {
  var a = "42";
  var b = "17";

  println(a == b); // should print "false"
  println(a != b); // should print "true"
  println(a < b);  // should print "false"
  println(a <= b); // should print "false"
  println(a > b);  // should print "true"
  println(a >= b); // should print "true"
  println(a + b);  // should print "4217"
  println(len(a)); // should print "2"
}
