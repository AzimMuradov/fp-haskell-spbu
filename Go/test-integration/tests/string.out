false
true
false
false
true
true
4217
2
